conda-envs/environment-test-py37.yml

```diff
@@ -17,7 +17,6 @@ dependencies:
 - libblas=*=*mkl
 - mkl-service
 - numpy>=1.15.0
-- pandas>=0.24.0
```

Comment:
Don't we have tests that cover pandas functionality? We would still want to test even if it's a user optional dependency